e2e/cypress/integration/enterprise/group_mentions/group_mentions_permissions_spec.js

```diff
@@ -134,7 +134,7 @@ describe('Group Mentions', () => {
             cy.get('#suggestionList').should('not.be.visible');
 
             // # Submit a post containing the group mention
-            cy.postMessage(`@${groupName}`);
+            cy.postMessage(`@${groupName} hello {enter}`);
```

Comment:
Shouldn’t be there a common function for post message with **@mention** ?

Reply:
Good point! Rather than introducing another command, I'm thinking to improve `cy.postMessage` in handling cases where autocomplete normally open up like at-mention, slash commands, etc. If it's fine with you, let's address it separately in another PR.

Reply:
> Good point! Rather than introducing another command, I'm thinking to improve `cy.postMessage` in handling cases where autocomplete normally open up like at-mention, slash commands, etc. If it's fine with you, let's address it separately in another PR.

I am Absolutely fine :)